main.py

```diff
@@ -126,11 +117,42 @@ def decodeLinkPath(path: str) -> str:
         return unixPath
 
     # Convert the path to Windows-style path for checking
-    windowsPath = path.replace('/', '\\')
+    windowsPath = path.replace("/", "\\")
     if pathExist(windowsPath):
         return windowsPath
 
-    return redirect(url_for('index')) # doesn't reload (TBI)
+    return redirect(url_for("index"))  # doesn't reload (TBI)
+
+def setup_logging() -> logging.Logger:
```

Comment:
following the naming convention `variableName` for both variables and functions declared by us.